nautobot_data_validation_engine/models.py

```diff
@@ -318,13 +319,13 @@ def clean(self):
 class DataCompliance(PrimaryModel):
     """Model to represent the results of an audit method."""
 
-    compliance_class_name = models.CharField(max_length=100, blank=False, null=False)
+    compliance_class_name = models.CharField(max_length=CHARFIELD_MAX_LENGTH, blank=False, null=False)
     last_validation_date = models.DateTimeField(blank=False, null=False, auto_now=True)
     content_type = models.ForeignKey(ContentType, on_delete=models.PROTECT, blank=False, null=False)
     object_id = models.CharField(max_length=200, blank=False, null=False)
```

Comment:
Should `object_id` and `validated_object_str` and `validated_attribute_value` also be increased? Honestly not clear on why these are limited at all.

Reply:
I have made the change for all the CharFields. As to why they are limited, I am not sure. Maybe to make it compatible with MySQL which does not have unlimited VARCHAR?